inc/template-functions.php

```diff
@@ -101,8 +102,10 @@ function wmf_get_role_hierarchy( $parent_id ) {
 	);
 
 	foreach ( $terms as $term_id => $parent ) {
-		if ( 0 < $parent ) {
+		if ( is_int( $parent ) && $parent > 0 ) {
 			$children[ $parent ][] = $term_id;
+		} else {
+			error_log( sprintf( 'Term ID %d has an invalid parent ID of [%s].', $term_id, var_export( $parent, true ) ) ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
```

Comment:
Ignore needs to be on the line above

Reply:
I added this error log to have some info about what is exactly causing the issue, as a way to uncover the root cause.